icd/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 # ~~~
 
-if (ANDROID)
+if (ANDROID OR MINGW)
```

Comment:
There is no point to building the mock driver for MINGW. It's not an official deliverable of this repo (See INSTALL_ICD).

If someone complains we can add it back in.